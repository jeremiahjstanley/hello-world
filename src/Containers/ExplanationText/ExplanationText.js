import React from 'react';
import { Link } from 'react-router-dom';


export const  ExplanationText = () => {

	return (

	    <div>
	    	<Link to='/' className='explanation-button'>â–²</Link>
	      <p className='explanation-paragraph'>
          The world is an awfully big place, and it seems to be 
          the focus of most conversations. It's the human condition 
          to wonder about what the day to day lives of other people 
          are like. Data gets a bad rap for being boring, but data 
          is fascinating, and I hope that Hello World can help 
          demystify data and help you understand the lump of 
          space rock hurdling through the cosmos that you 
          currently call home.
	      </p>
	    </div>
         
	);
};


export default ExplanationText;