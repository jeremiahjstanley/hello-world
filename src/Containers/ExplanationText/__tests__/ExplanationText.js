import React from 'react';
import { shallow } from 'enzyme'
import { ExplanationText } from '../ExplanationText';


describe('ExplanationText', () => {

  let wrapper;

  beforeEach(() => {

    wrapper = shallow(<ExplanationText />);

  });

	it('should match the snapshot', () => {

		expect(wrapper).toMatchSnapshot();

	});

});