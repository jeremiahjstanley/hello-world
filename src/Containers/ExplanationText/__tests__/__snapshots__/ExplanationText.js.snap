// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExplanationText should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Link
          className="explanation-button"
          replace={false}
          to="/"
>
          ▲
</Link>,
        <p
          className="explanation-paragraph"
>
          The world is an awfully big place, and it seems to be the focus of most conversations. It's the human condition to wonder about what the day to day lives of other people are like. Data gets a bad rap for being boring, but data is fascinating, and I hope that Hello World can help demystify data and help you understand the lump of space rock hurdling through the cosmos that you currently call home.
</p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "▲",
          "className": "explanation-button",
          "replace": false,
          "to": "/",
        },
        "ref": null,
        "rendered": "▲",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "The world is an awfully big place, and it seems to be the focus of most conversations. It's the human condition to wonder about what the day to day lives of other people are like. Data gets a bad rap for being boring, but data is fascinating, and I hope that Hello World can help demystify data and help you understand the lump of space rock hurdling through the cosmos that you currently call home.",
          "className": "explanation-paragraph",
        },
        "ref": null,
        "rendered": "The world is an awfully big place, and it seems to be the focus of most conversations. It's the human condition to wonder about what the day to day lives of other people are like. Data gets a bad rap for being boring, but data is fascinating, and I hope that Hello World can help demystify data and help you understand the lump of space rock hurdling through the cosmos that you currently call home.",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Link
            className="explanation-button"
            replace={false}
            to="/"
>
            ▲
</Link>,
          <p
            className="explanation-paragraph"
>
            The world is an awfully big place, and it seems to be the focus of most conversations. It's the human condition to wonder about what the day to day lives of other people are like. Data gets a bad rap for being boring, but data is fascinating, and I hope that Hello World can help demystify data and help you understand the lump of space rock hurdling through the cosmos that you currently call home.
</p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "▲",
            "className": "explanation-button",
            "replace": false,
            "to": "/",
          },
          "ref": null,
          "rendered": "▲",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "The world is an awfully big place, and it seems to be the focus of most conversations. It's the human condition to wonder about what the day to day lives of other people are like. Data gets a bad rap for being boring, but data is fascinating, and I hope that Hello World can help demystify data and help you understand the lump of space rock hurdling through the cosmos that you currently call home.",
            "className": "explanation-paragraph",
          },
          "ref": null,
          "rendered": "The world is an awfully big place, and it seems to be the focus of most conversations. It's the human condition to wonder about what the day to day lives of other people are like. Data gets a bad rap for being boring, but data is fascinating, and I hope that Hello World can help demystify data and help you understand the lump of space rock hurdling through the cosmos that you currently call home.",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
